lua/rest-nvim/utils/init.lua

```diff
@@ -48,27 +48,65 @@ end
 -- variables
 M.read_env_file = function()
```

Comment:
Name of `read_env_file` should be changed something like `get_variables` as it no longer just reads env file.